extractor/src/main/java/com/example/extractor/CheckstyleExampleExtractor.java

```diff
@@ -147,6 +172,19 @@ public static void main(final String[] args) throws Exception {
             // Output default projects list
             outputDefaultProjectsList(projectsOutputPath);
         }
+        else if (args.length == CONFIG_FILE_ARG_COUNT
+                && "--config-file".equals(args[CONFIG_FILE_FLAG_INDEX])) {
```

Comment:
It should not be any different from regular execution on Input.
We should have a some structure inside that describe all Checks properties that having external property files. We do not have much of them, better to hard code them for now.

We might need to simply update examples in main repository to reference same config name and use the same system variable and path in config.
So we can simply put a predefined file to generation target folder.

If some module define content of config and we need to use it, we can rename them to be predictable and just use specific logic for module that is registered as external file module.

Reply:
https://checkstyle.org/checks/header/regexpheader.html#Example2-config

We can make it:
```
<module name="Checker">
  <module name="RegexpHeader">
    <property name="headerFile" value="${config.path}/java.header"/>
    <property name="multiLines" value="10, 13"/>
  </module>
</module>
```
in main repository.

And in test-configs project just parse Input as regular Input, if we detect special module like this, we copy config files  that are hard coded path to them in our code structure.

During execution of test bundle we just always define this system variable, even it will not be in use by module.

Reply:
@relentless-pursuit , ping. Do you need any assistance?

Reply:
> @relentless-pursuit , ping. Do you need any assistance?

@romani, can you re-review?